README.md

```diff
@@ -128,8 +141,17 @@ All props accept arrays as values for mobile-first responsive styles.
 
 styled-system includes a higher order component to add style props to any component.
 
+**With `styled-components`**
+
+```jsx
+import hoc from 'styled-system/dist/styled-components/hoc'
```

Comment:
Yeah, that was the intent. If people using other CSS-in-JS libs (glamorous, aphrodite, glamor, jss, etc.) they'll not want to load `styled-components` to use `styled-system`.